apollo-router/src/configuration/metrics.rs

```diff
@@ -144,7 +144,9 @@ impl Metrics {
             value.apollo.router.config.authorization,
             "$.authorization",
             opt.require_authentication,
-            "$[?(@.require_authentication == true)]"
+            "$[?(@.require_authentication == true)]",
+            opt.preview_directives,
```

Comment:
So far we've not been putting config metrics under experimental or preview. That way when this graduates we don't have to make changes to dashboards

```suggestion
            opt.directives,
```